internal/operator/backrest/restore.go

```diff
@@ -35,6 +36,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// restoreTargetRegex defines a regex to detect if a restore target has been specified
+// for pgBackRest using the '--target' option
+var restoreTargetRegex = regexp.MustCompile("--target(=| +)")
```

Comment:
any reason why it's not `\s+`? I did a few double takes on that.

Reply:
No specific reason, just different syntax.

Reply:
In this context, `\s` seems more appropriate. We could go the other direction and match `--target[^-]`, but it doesn't feel optimal either.

Reply:
Also, was really just looking to match on a spaces, and not necessarily also on tabs and newlines (not that those should really ever be in there).